presto-docs/src/main/sphinx/functions/ml.rst

```diff
@@ -0,0 +1,151 @@
+===================
+Machine Learning Functions
+===================
+
+The machine learning plugin in Presto provides machine learning functionality
+as an aggregation function. It enables us to train Support Vector Machine (SVM)
+based classifier and regressor for the supervised learning problems.
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+.. note::
+  These machine learning functions are not fully optimized in a distributed manner.
```

Comment:
The machine learning functions are not optimized for distributed processing. The capability to train large data sets is limited by this execution of the final training on a single instance.

Reply:
Also ... ideally keep it to 80 char line length